csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1621,9 +1621,25 @@ Example:
      Siemens ProductCERT
 ```
 
-##### 3.2.1.8.5 Document Property - Publisher - Vendor ID
+##### 3.2.1.8.5 Document Property - Publisher - Namesapce
+
+The Namespace of publisher (`namesapce`) of value type `string` and format `uri` contains a URL which is under control of the issuing party and can be used as a globally unique identifier for that issuing party. The URL SHALL be normalized.
```

Comment:
You should consider how this identity binds to a signing identity.  What happens when these things radically differ?  Which do you trust?  As a matter of tooling I am likely to discard this field for any automated processing unless the the document is secured in some way.

Reply:
Isn't this true for the entire document? I vaguely recall XML DSIG was discussed but I don't see a digital signature specification/requirement for CVRF. Maybe the decision was to rely on transport layer security? See: https://github.com/oasis-tcs/csaf/issues/241.

Reply:
The real question raised here is: Do we allow the namespace to be
1. only a URL
2. only a email
3. or both?

I would like to continue this discussion at #275